dev/benchmarks/complex_layout/android/app/src/main/AndroidManifest.xml

```diff
@@ -3,12 +3,12 @@ Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file. -->
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.yourcompany.complexLayout">
+    package="dev.benchmarks.complex_layout">
 
     <uses-permission android:name="android.permission.INTERNET"/>
 
     <application android:name="io.flutter.app.FlutterApplication" android:label="complex_layout" android:icon="@mipmap/ic_launcher">
-        <activity android:name="io.flutter.app.FlutterActivity"
+        <activity android:name=".DummyPlatformViewActivity"
```

Comment:
I think we should avoid touching this file right now - it would probably be better to just create a new benchmark for this rather than touching the existing one, since this is used by several other benchmarks.

I'm particularly worried about what happens if this puts a devicelab device into a bad state right now.

/cc @godofredoc